docs/book/setup/upgrading-sylius.rst

```diff
@@ -1,8 +1,5 @@
-.. index::
-   single: Upgrading
-
-Upgrading
-=========
+How to Upgrade Sylius Application?
```

Comment:
:(

Reply:
As mentioned above, I'll revert this change. However, "Upgrading" doesn't say much. I'll go with "Upgrading Sylius Application" - it states its purpose better.